recipes/yasm/meta.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: yasm
```

Comment:
I guess this isn't supposed to be built on Windows? If so, just add a section which does ```build/skip: True  # [win]```